Mathlib/Topology/ContinuousFunction/Compact.lean

```diff
@@ -330,6 +330,17 @@ theorem dist_lt_of_dist_lt_modulus (f : C(α, β)) (ε : ℝ) (h : 0 < ε) {a b
     (w : dist a b < f.modulus ε h) : dist (f a) (f b) < ε :=
   (Classical.choose_spec (uniform_continuity f ε h)).2 w
 
+/-- A family of continuous maps `f : α → C(β, γ)` is continuous on an open set if it is
+  pointwise continuous, provided that `β` is compact and `α` is locally compact. -/
+lemma continuousOn_of_pointwise_continuousOn_of_compact {α : Type*} {β : Type*} {γ : Type*}
+    [UniformSpace α] [UniformSpace β] [LocallyCompactSpace α] [CompactSpace β] [UniformSpace γ]
```

Comment:
Is it true for topological spaces?

Reply:
I don't know. If it is I have no idea how to prove it.

Reply:
It is not important that `α` be a uniform space, because it the continuity is only used locally, and it is locally compact, and compact spaces have a canonical uniform structure.

Reply:
Same for `β`.

Reply:
I see, thanks!